ElectronClient/app/gui/NoteText.jsx

```diff
@@ -1193,6 +1193,7 @@ class NoteTextComponent extends React.Component {
 			type: 'WINDOW_COMMAND',
 			name: 'setTags',
 			noteId: this.state.note.id,
+			pendingTag: this.state.note.pendingTag || []
```

Comment:
Should be plural if it's an array.

Reply:
I think/assume that very soon someone (me) will want to have multiple tags planned on a new note.  Based that and on your comments about "pending" I think I should:
- change "pendingTag" to "plannedTags" everywhere.
-  and consistently use an array to pass the plannedTags.

Any testing ideas?